flybywiresim/a32nx
# fix: rudder trim text position

## Summary of Changes
The following PR fixes the position for the Rudder Trim display.
##  Rudder Trim
![image](https://user-images.githubusercontent.com/43104362/106394329-c6a0b600-6404-11eb-9eaf-2f9adbfae254.png)


Discord Username: `Snapmatics#0001`

## Testing Instructions

- Turn ON external power
- Check if the Rudder Trim text position is fixed


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link
